src/redux/features/books/booksSlice.js

```diff
@@ -1,48 +1,72 @@
-import { createSlice } from '@reduxjs/toolkit';
+import { createSlice, createAsyncThunk } from '@reduxjs/toolkit';
+import axios from 'axios';
+
+const url = 'https://us-central1-bookstore-api-e63c8.cloudfunctions.net/bookstoreApi/apps/';
+const urlId = 'UzyepzOzvkbNU3PJxe1b';
+
+export const getBooks = createAsyncThunk('books/getBooks', async () => {
+  const response = await axios.get(`${url}${urlId}/books`);
+
+  const books = Object.entries(response.data).map((item) => ({
+    ...item[1][0],
+    item_id: item[0],
+  }));
+  // console.log('get', books);
+  return books;
+});
+
+export const addBook = createAsyncThunk('books/addBook', async (book) => {
+  try {
+    await axios.post(`${url}${urlId}/books`, book);
+  } catch (error) {
+    throw new Error(error);
+  }
+  // console.log('post', book);
```

Comment:
- [ ] Nice job using try and catch with API calls, but please make your code clean of any commented(unused) code. (don't forget to remove console logs after you have done with them)